tests/json_queries.cc

```diff
@@ -103,6 +103,43 @@ TEST(AddImage, simpleAdd)
     PMGDQueryHandler::destroy();
 }
 
+TEST(FindImage, simpleFind)
```

Comment:
when doing query-response testing, using Python is best. We are already testing this in python.